.github/workflows/check-service-labels.yml

```
@@ -4,13 +4,14 @@ permissions:
   pull-requests: read
 
 on:
-  pull_request:
-    types: [opened, edited, reopened, labeled, unlabeled, synchronize]
+  pull_request_target:
+    types: [labeled, unlabeled]
```

I'll expand this to include opened and reopened. I noticed the leg gets stuck in those cases.

```suggestion
    types: [opened, reopened, labeled, unlabeled]
```